src/olympia/activity/tests/test_utils.py

```diff
@@ -14,25 +14,99 @@
 from olympia.amo.urlresolvers import reverse
 from olympia.activity.models import ActivityLogToken, MAX_TOKEN_USE_COUNT
 from olympia.activity.utils import (
-    add_email_to_activity_log, log_and_notify, send_activity_mail,
-    ActivityEmailParser)
+    add_email_to_activity_log, add_email_to_activity_log_wrapper,
+    log_and_notify, send_activity_mail, ActivityEmailEncodingError,
+    ActivityEmailParser, ActivityEmailTokenError, ActivityEmailUUIDError)
 from olympia.devhub.models import ActivityLog
 
 
 TESTS_DIR = os.path.dirname(os.path.abspath(__file__))
-sample_message = os.path.join(TESTS_DIR, 'emails', 'message.json')
+sample_message_file = os.path.join(TESTS_DIR, 'emails', 'message.json')
+sample_message_content = json.loads(open(sample_message_file).read())
```

Comment:
In theory...

```python
with open(sample_message_file) as fobj:
    sample_message_content = json.loads(fobj.read())
```

is a bit safer and closes the file object correctly. (your code leaves it open unfortunately - which isn't a problem here but can lead to weird exception or hangs with pytest and weird io errors)